src/pages/CommunityResourcesPage/ui/CommunityResourcesPage.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from "react";
+import ResourceCard from "../../../shared/ui/ResourceCard/ResourceCard";
+import { resources } from "./constants";
+import styles from "./CommunityResourcesPage.module.css";
+import Button from "../../Pomodoro/components/Button/Button";
+
+const CommunityResources: React.FC = () => {
+  const [resourcesList] = useState(resources);
+  const [selectedTags, setSelectedTags] = useState<string[]>(["All"]); // Изначально All выбран
+  const [showAllTags, setShowAllTags] = useState(false); // Стейт для управления показом всех тегов
+
+  const handleTagClick = (tag: string) => {
+    if (tag === "All") {
+      if (selectedTags.includes("All")) {
+        setSelectedTags([]);
+      } else {
+        setSelectedTags(["All"]);
+      }
+    } else {
+      if (selectedTags.includes("All")) {
+        setSelectedTags([tag]);
+      } else {
+        if (selectedTags.includes(tag)) {
+          setSelectedTags(
+            selectedTags.filter((selectedTag) => selectedTag !== tag)
+          );
+        } else {
+          setSelectedTags([...selectedTags, tag]);
+        }
+      }
+    }
+  };
+
+  const filteredResources = resourcesList.filter((resource) => {
+    if (selectedTags.length === 0 || selectedTags.includes("All")) return true;
+    return selectedTags.some((tag) => resource.tags.includes(tag));
+  });
+
+  const handleAddToFavorites = (id: number) => {
```

Comment:
We don't currently have this feature of adding resources to favorites in MVP :)

Reply:
> We don't currently have this feature of adding resources to favorites in MVP :)
I thought "The resources are also displayed on the user's page." task means that the user can see saved resources on his page. I'll remove it, okay)